### PR TITLE
Fixing case sensitivity and grammar/orthography

### PR DESCRIPTION
**Pull request by HackintoshHD**
[Disclaimer: I hope this is the right file for the commit - I'm not familiar with the Transifex interaction with GitHub.]

There is currently a total of three different versions spelling the app's name "PassMan" in the strings for localisation on Transifex: "Passman", "PassMan" and "passman". Assuming the intended one is the one with the middle upper case "M", this pull request would correct the spelling to "PassMan". The other changes concern grammar ("passwords" in plural) and orthography ("this is", not "this if").

### CODE REVIEW

File: templates/main.php [hunk 1]
```diff
@@ -1187,7 +1187,7 @@ class="otpSecret"/></label>
 
 <div id="encryptionKeyDialog" style="display: none;">
   <p><?php p($l->t('Enter your encryption key.')); ?></p>
-  <p><?php p($l->t('If this if the first time you use Passman, this key will be used for encryption your
+  <p><?php p($l->t('If this is the first time you use PassMan, this key will be used for encryption of your
```
Comment (DeepDiver1975) [inline templates/main.php]: I really wonder if that line break is such a great idea ...

File: templates/main.php [hunk 2]
```diff
@@ -1187,7 +1187,7 @@ class="otpSecret"/></label>
 
 <div id="encryptionKeyDialog" style="display: none;">
   <p><?php p($l->t('Enter your encryption key.')); ?></p>
-  <p><?php p($l->t('If this if the first time you use Passman, this key will be used for encryption your
```
Comment (brantje) [inline templates/main.php]: Its Passman

File: templates/main.php (hunk 1 above)
Comment (brantje) [inline templates/main.php]: Its Passman

Comment (HackintoshHD) [inline templates/main.php]: Okay, if you want it to be "Passman", then I'll start over again and create a new pull request.

**Comment by HackintoshHD** [comment]
@brantje I've now unified the app's name to "Passman" without the middle upper case "M" as indicated [above](https://github.com/brantje/passman/pull/92#discussion_r23519905). I've also removed the line break in line 1190 of main.php @DeepDiver1975 was referring to (I think).

**Comment by brantje** [comment]
Nice, thanks for your input.

**Pull request merged**